arangod/Cache/Manager.cpp

```diff
@@ -338,11 +357,14 @@ std::tuple<bool, Metadata, std::shared_ptr<Table>> Manager::registerCache(
   }
 
   if (ok) {
-    metadata =
-        Metadata(Cache::kMinSize, fixedSize, table->memoryUsage(), maxSize);
-    ok = increaseAllowed(metadata.allocatedSize - table->memoryUsage(), true);
+    std::uint64_t memoryUsage = table->memoryUsage();
+    metadata = Metadata(Cache::kMinSize, fixedSize, memoryUsage, maxSize);
+    TRI_ASSERT(metadata.allocatedSize >= memoryUsage);
```

Comment:
is equal ok?
It looks unnecessary to call other part of code in such case

Reply:
`metadata.allocatedSize` will probably always be bigger than `memoryUsage` here. But this is an implementation detail.
But for the assertions that care about `_globalAllocation` and `_fixedAllocation` this should not matter too much. For these two, we always (and only) want to check the invariant that `_globalAllocation` is always >= `_fixedAllocation`.